records/state/state.go

```diff
@@ -46,9 +46,22 @@ type Label struct {
 
 // Status holds a task status as defined in the /state.json Mesos HTTP endpoint.
 type Status struct {
-	Timestamp float64 `json:"timestamp"`
-	State     string  `json:"state"`
-	Labels    []Label `json:"labels,omitempty"`
+	Timestamp       float64         `json:"timestamp"`
+	State           string          `json:"state"`
+	Labels          []Label         `json:"labels,omitempty"`
+	ContainerStatus ContainerStatus `json:"container_status,omitempty"`
+}
+
+// ContainerStatus holds container metadata as defined in the /state.json
+// Mesos HTTP endpoint.
+type ContainerStatus struct {
+	NetworkInfos []NetworkInfo `json:"network_infos,omitempty"`
+}
+
+// NetworkInfo holds a network address resolution as defined in the
+// /state.json Mesos HTTP endpoint.
+type NetworkInfo struct {
+	IPAddress string `json:"ip_address,omitempty"`
```

Comment:
How do we know if the `IPAddress` is v4 or v6? Are all clients supposed to try parsing one and then fallback?